lua/starfall/libs_sv/prop.lua

```diff
@@ -603,6 +603,10 @@ function props_library.createSent(pos, ang, class, frozen, data)
 			end
 
 			entity = duplicator.CreateEntityFromTable(ply, enttbl)
+			if not isentity(entity) then
+				entity = nil
+				error("Factory func returned invalid value")
```

Comment:
Should throw at level `2` here, otherwise file and line of this error will be included in the final message:
`SF:main:11: Failed to create entity (starfall/libs_sv/prop.lua:608: Factory func returned invalid value)`.